From f5285a88d1e673c3c14d16d5ab298ef70ef91f25 Mon Sep 17 00:00:00 2001
From: Pankit Garg <pankit.garg@nxp.com>
Date: Tue, 16 Jan 2018 10:42:43 +0530
Subject: [PATCH 105/129] armv8: ls1088ardb: secure boot: Use tiny printf to save spl size

Using tiny printf remove format specifier debug capabilities for
secure boot defconfig

Signed-off-by: Pankit Garg <pankit.garg@nxp.com>
---
 .../ls1088ardb_sdcard_qspi_SECURE_BOOT_defconfig   |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/configs/ls1088ardb_sdcard_qspi_SECURE_BOOT_defconfig b/configs/ls1088ardb_sdcard_qspi_SECURE_BOOT_defconfig
index 5f02fea..6158fc7 100644
--- a/configs/ls1088ardb_sdcard_qspi_SECURE_BOOT_defconfig
+++ b/configs/ls1088ardb_sdcard_qspi_SECURE_BOOT_defconfig
@@ -46,3 +46,4 @@ CONFIG_RSA=y
 CONFIG_SPL_RSA=y
 CONFIG_SPL_CRYPTO_SUPPORT=y
 CONFIG_SPL_HASH_SUPPORT=y
+CONFIG_USE_TINY_PRINTF=y
-- 
1.7.1

